epix/quanta_generation.py

```diff
@@ -32,19 +32,6 @@ def quanta_from_NEST(en, model, e_field, A, Z, create_s2, **kwargs):
     nc = nestpy.NESTcalc(nestpy.VDetector())
     density = 2.862  # g/cm^3
 
-    # Fix for Kr83m events.
-    # Energies have to be exactly 32.1 keV or 9.1 keV
-    # See: https://github.com/NESTCollaboration/nest/blob/master/src/NEST.cpp#L567
-    # and: https://github.com/NESTCollaboration/nest/blob/master/src/NEST.cpp#L585
-    # TODO: There must be something odd with our clustering, that we need
-    #  this?
-    max_allowed_energy_difference = 1  # keV
-    if model == 11:
-        if abs(en - 32.1) > max_allowed_energy_difference:
-            en = 32.1
-        if abs(en - 9.1) > max_allowed_energy_difference:
-            en = 9.1
-
```

Comment:
See Sophia's message here: https://github.com/XENONnT/epix/issues/17#issuecomment-832656818

I just read through it and I don't completely understand if this was an issue back then. Are we missing Kr83m events because of clustering shifting/cutting our energies?

Reply:
Hi Diego, thanks for your checks. Let me try to describe the problem we had some time ago and why we can now remove the lines you commented.

In previous NEST versions the Kr83m model was only working like intended wenn the input energies were exactly 32.1 and/or 9.4 keV. Since this is basically never the case in our simulations, these lines "rounded" all energies on the values needed by NEST. In the issue you linked was described that there is a mistake in the Kr83m energy. Of course it should be 9.4 and not 9.1 keV. This means that for Kr83m simulations at that time we would call the Kr83m model with a wrong energy and the number of photons and electrons should be a bit off. 

In the recent NEST versions a functionality was added that gets rid of the energy equal x requirement. If I got it right the energy must now be only close to 32.1 and/or 9.4 keV in order to trigger the right case in NEST.

Reply:
Perfect! Thanks a lot for the explanation. I guess @sophiafarrell is then happy with deleting this part (sorry, forgot to tag her before).

Reply:
Yep! I should say that there is an argument in NEST, some minimum allowable energy difference, but that this is very small, like order 1e-9 I think? So unless this is valid in how close you can get, I would maybe keep it as is. Or the more elegant thing to do would be to pass the right minimum difference into NEST. I can dig that up if you want to go that route, but my suggestion would be to leave as is if you need this high of a margin. Still though, change to 9.4 keV though, and not 9.1;)

Reply:
(See here for code) https://github.com/NESTCollaboration/nest/blob/5d01f09228d04459976e2f52a0ffb15f3fd3e9f0/src/ValidityTests.cpp#L6
and we can hack up NEST such that this is easy for you to pass through to NESTcalc (or in nestpy, too) if needed. But that will take time since NEST just got updated last week...

Reply:
Ahh okay, 1e-9 is way to small I think. I will bring the lines back (with the energy fix of course) and we should be good to go. Thanks for commenting Sophia.

Reply:
Done in https://github.com/XENONnT/epix/pull/30/commits/9468d43c5b8837002d076acc355628a611b3d71d